src/Card/index.jsx

```diff
@@ -1,12 +1,62 @@
-import Card from 'react-bootstrap/Card';
+import React from 'react';
+import BaseCard from 'react-bootstrap/Card';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import CardContext, { CardContextProvider } from './CardContext';
 import CardHeader from './CardHeader';
+import CardDivider from './CardDivider';
+import CardSection from './CardSection';
+import CardFooter from './CardFooter';
+import CardImageCap from './CardImageCap';
+import CardBody from './CardBody';
+
+const Card = React.forwardRef(({
+  orientation,
+  className,
+  isClickable,
+  ...props
+}, ref) => (
+  <CardContextProvider orientation={orientation}>
+    <BaseCard
+      {...props}
+      className={classNames(className, 'pgn__card', {
+        horizontal: orientation === 'horizontal',
+        clickable: isClickable,
```

Comment:
With the `isClickable` prop, the card image overlaps the border of the card itself, which looks a bit odd.

It also seems that I can focus on a card to get the box-shadow and border even without the `isClickable` prop.

![image](https://user-images.githubusercontent.com/2828721/147260683-79798f3c-33ec-4543-91ae-6235155d8f03.png)

Reply:
Yeah, the styles were a bit off... Updated them, should be good now (although for images with dark background the border my not be very noticeable after focus), thank you.